packages/dx-react-grid-export/src/types/export/exporter.types.ts

```diff
@@ -49,4 +51,7 @@ export type ExporterState = {
   selectedOnly: boolean;
 };
 
-export declare const GridExporter: React.ComponentType<ExporterProps>;
+export declare const GridExporter: React.ComponentType<ExporterProps> & {
+  /** A method that exports data. */
+  exportGrid: (options?: any) => void;
```

Comment:
Method `startExport` calls `exportGrid` and provides `options: object`. Should we change this type?
```ts
exportGrid: (options?: object) => void;
```